### ISSUE TITLE
Chart Tab: Monthly Totals 

### ISSUE THREAD

**Issue by JUG1T3R** [NonlinearFruit/Retrospector#81]
In an effort to make months more comparable and create a more standard size for the Monthly Totals chart, I think it would be useful if all 12 months were displayed once on the x axis. Then in different colors you could display the different year's totals for each month. You would need a legend to decipher which year is which.
It would be cool to be able to see more easily how your monthly media consumption compares year to year to look for trends.

**Comment by NonlinearFruit** [comment]
I like this idea. This would be a good additional chart (I'm fond of the current Monthly Totals for seeing how my consumption changes of time). Any ideas on how to pack in more charts?

If I was going to remove charts, I'd probably ditch the Factoids section. I use it a little but I use everything else a lot more and I've never used the stats in the factoid section :(

**Comment by JUG1T3R** [comment]
Yeah I was going to mention to you that I never really ever used either of the factoid charts either. I wouldn't miss them if they were gone.

**Comment by NonlinearFruit** [comment]
I've created an issue for discussing the factoid section #82
